### PR TITLE
FEATURE: SD-476 - Disable Headline exporting

### PR DESCRIPTION
**Pull request by ptquang86**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)
- [x] Squash commits

# Related PRs
- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2287
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2014

# Related Jira tasks
- [SD-476](https://jira.intgdc.com/browse/SD-476)

### CODE REVIEW

**Comment by ptquang86** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3901/ : FAILURE in 4m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4938/ : SUCCESS in 11s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2133/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1481/ : SUCCESS in 13m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4939/ : SUCCESS in 6s

**Comment by ptquang86** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2134/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1482/ : SUCCESS in 13m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4941/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3903/ : SUCCESS in 9m 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4940/ : SUCCESS in 7m 41s

**Comment by ptquang86** [comment]
@gooddata/react-components@7.1.0-ptquang86-qp-sd-476-headline-2019-06-05T06-29-48-120Z

**Comment by ptquang86** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1023.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1483/ : SUCCESS in 31m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4943/ : SUCCESS in 13s

**Comment by naibinh** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3948/ : SUCCESS in 4m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5000/ : SUCCESS in 7s

File: src/internal/constants/uiConfig.ts [hunk 1]
```diff
@@ -61,6 +61,14 @@ export const defaultFilters = {
     },
 };
 
+export const disabledExportResultConfig = {
+    exportResult: { supported: false },
```
Comment (nadtan) [inline src/internal/constants/uiConfig.ts]: I would rename it to `export: { supported: true }` cause exportResult has different meaning from its purpose

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3957/ : SUCCESS in 8m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5010/ : SUCCESS in 8s

**Comment by naibinh** [comment]
LGTM

File: src/internal/constants/uiConfig.ts (hunk 1 above)
Comment (ptquang86) [reply src/internal/constants/uiConfig.ts]: updated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3968/ : SUCCESS in 7m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5022/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2163/ : SUCCESS in 12m 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3969/ : SUCCESS in 15m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1044/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1208/ : SUCCESS in 17m 38s

**Pull request merged**